auth_backends/backends.py

```diff
@@ -99,6 +106,169 @@ def _map_user_details(self, response):
         return dest
 
 
+class EdXOpenIdConnectMultiSite(OpenIdConnectAuth):
+    name = 'edx-oidc-multisite'
+
+    ACCESS_TOKEN_METHOD = 'POST'
+    REDIRECT_STATE = False
+    ID_KEY = 'preferred_username'
+
+    DEFAULT_SCOPE = ['openid', 'profile', 'email'] + getattr(settings, 'EXTRA_SCOPE', [])
+
+    PROFILE_TO_DETAILS_KEY_MAP = {
+        'preferred_username': u'username',
+        'email': u'email',
+        'name': u'full_name',
+        'given_name': u'first_name',
+        'family_name': u'last_name',
+        'locale': u'language',
+    }
+
+    auth_complete_signal = django_dispatch.Signal(providing_args=["user", "id_token"])
+
+    def id_token_issuers(self):
+        return self.setting('ISSUERS', None)
+
+    def authorization_url(self):
```

Comment:
I will eventually file a PR against omab/python-social-auth to use these methods instead of the default class attributes.